types/xml2js/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for node-xml2js 0.4
+// Type definitions for xml2js 0.4
```

Comment:
Changing to `xml2js` to match with the folder name.

I think `node-xml2js` is confusing (although the git repo is titled like that), since the package is available on NPM as:
- https://www.npmjs.com/package/xml2js
- https://www.npmjs.com/package/@types/xml2js